edxval/api.py

```diff
@@ -143,6 +143,55 @@ def update_video_status(edx_video_id, status):
     video.save()
 
 
+def get_organization_transcript_credentials_state(org, provider=None):
+    """
+    Returns transcript credentials state for a course organization.
+
+    Arguments:
+        org (unicode): course organization
+        provider (unicode): transcript provider
+
+    Returns:
+        dict: contains the information regarding organization's transcript provider credentials
+
+    Returned dict will look like one of below
+    {
+        'org': 'edX',
+        'Cielo24': True,
+    }
+
+    {
+        'org': 'edX',
+        'Cielo24': False,
+        '3PlayMedia': True,
+    }
+    """
+    filter_attrs = {'org': org}
+    if provider:
+        filter_attrs['provider'] = provider
+
+    credentials_state = OrganizationTranscriptCredentialsState.objects.filter(**filter_attrs)
+    if credentials_state.exists():
+        credentials_state_info = {}
+        for credential_state in credentials_state:
+            credentials_state_info['org'] = credential_state.org
+            credentials_state_info[credential_state.provider] = credential_state.exists
+
+        return credentials_state_info
+
+
+def update_organization_transcript_credentials_state(org, provider, exists):
```

Comment:
I would rename it to `update_transcript_organization_credentials_state`

Reply:
`update_3rd_party_transcript_credentials_state_for_org`